CRUD-App/routes/api.php

```diff
@@ -18,3 +18,17 @@
     return $request->user();
 });
 
+//Display Data 
+Route::get('display/{id?}',[ProductController::class,"display"] );
```

Comment:
@Yashbanu why a question mark after Id ! any purpose ?

Reply:
@Naveen1895 instead of providing two different route i.e with parameter and without parameter , I used "?" to indicate that the parameter is optional

Reply:
@Yashbanu In that case we actually create two different routes.. one as `/display `and another as `/display/{id}`